operator/internal/manifests/internal/config/loki-config.yaml

```
@@ -107,6 +107,16 @@ compactor:
 {{- end }}
   delete_request_store: {{.ObjectStorage.SharedStore}}
 {{- end }}
+{{- if $l := .Stack.Limits.Global }}
+{{- with $l.OTLP }}
+distributor:
+  otlp_config:
+    default_resource_attributes_as_index_labels:
+    {{- range .IndexedResourceAttributes }}
+    - {{ . }}
+    {{- end }}
+{{- end }}
```

I'd suggest to wrap this in another `with` to avoid producing an empty section when no `IndexesResourceAttributes` are present:

```suggestion
{{- with $l.OTLP }}{{- with .IndexedResourceAttributes }}
distributor:
  otlp_config:
    default_resource_attributes_as_index_labels:
    {{- range . }}
    - {{ . }}
    {{- end }}
{{- end }}{{- end }}
```